### PR TITLE
[CPU][I64] Transpose node support.

### PR DESCRIPTION
**Pull request by nshchego**
### Details:
- *Int64 precision support for Transpose node*
- *Some general test changes for the following nodes.*

### Tickets:
 - **

### CODE REVIEW

File: src/inference/dev_api/cpp_interfaces/interface/ie_internal_plugin_config.hpp [hunk 1]
```diff
@@ -110,6 +110,11 @@ INFERENCE_ENGINE_1_0_DEPRECATED DECLARE_CONFIG_VALUE(ENABLE);
 INFERENCE_ENGINE_1_0_DEPRECATED DECLARE_CONFIG_VALUE(IGNORE_CALLBACK);
 INFERENCE_ENGINE_1_0_DEPRECATED DECLARE_CONFIG_VALUE(DISABLE);
 
+/**
+ * @brief Enables inference with INT64 data type in CPU plugin if it's presented in the original model.
+ */
+DECLARE_CONFIG_KEY(CPU_NATIVE_I64);
```
Comment (ilya-lavrenov) [inline src/inference/dev_api/cpp_interfaces/interface/ie_internal_plugin_config.hpp:116]: who is the user of this private property?

**Review by v-Golubev — APPROVED** [review]
Only minor comments from my side, in general LGTM

File: src/inference/dev_api/cpp_interfaces/interface/ie_internal_plugin_config.hpp (hunk 1 above)
Comment (nshchego) [reply src/inference/dev_api/cpp_interfaces/interface/ie_internal_plugin_config.hpp:116]: This allows to enable I64 execution in tests for now.

File: src/plugins/intel_cpu/src/nodes/executors/common/ref_opt_transpose.cpp [hunk 2]
```diff
@@ -124,15 +124,16 @@ void RefOptimizedTransposeExecutor::exec(const std::vector<MemoryCPtr>& src, con
...
 
 bool RefOptimizedTransposeExecutor::init(const TransposeParams &transposeParams,
-                                const std::vector<MemoryDescPtr> &srcDescs,
-                                const std::vector<MemoryDescPtr> &dstDescs,
-                                const dnnl::primitive_attr &attr) {
-    return true;
+                                         const std::vector<MemoryDescPtr> &srcDescs,
+                                         const std::vector<MemoryDescPtr> &dstDescs,
+                                         const dnnl::primitive_attr &attr) {
+return true;
```
[the remark this answers is not in the document]
Comment (nshchego) [reply src/plugins/intel_cpu/src/nodes/executors/common/ref_opt_transpose.cpp]: Done

File: src/plugins/intel_cpu/src/nodes/transpose.cpp [hunk 3]
```diff
@@ -20,13 +17,12 @@ namespace node {
 
 bool Transpose::isSupportedOperation(const std::shared_ptr<const ov::Node>& op, std::string& errorMessage) noexcept {
     try {
-        if (!one_of(op->get_type_info(),
-                ov::op::v1::Transpose::get_type_info_static())) {
+        if (!one_of(op->get_type_info(), op::v1::Transpose::get_type_info_static())) {
```
[the remark this answers is not in the document]
Comment (nshchego) [reply src/plugins/intel_cpu/src/nodes/transpose.cpp]: Done

**Pull request closed without merging**